pkgs/development/python-modules/mujoco/default.nix

```diff
@@ -24,13 +26,19 @@ buildPythonPackage rec {
   # in the project's CI.
   src = fetchPypi {
     inherit pname version;
-    hash = "sha256-rZNVihIuvNJnQWqA5tV9DG5r3/LttWNW6fN2js+fDb8=";
+    hash = "sha256-ESEnPeL79O0wnllEo9s50B84WyINIOeMRg7E78BpRbM=";
   };
 
   nativeBuildInputs = [ cmake setuptools ];
   dontUseCmakeConfigure = true;
   buildInputs = [ mujoco pybind11 ];
-  propagatedBuildInputs = [ glfw numpy ];
+  propagatedBuildInputs = [
+    absl-py
+    etils
+    glfw
+    numpy
+    pyopengl
+  ];
```

Comment:
These inputs aren't required for the new patch version. Why the addition?

Reply:
When I tried building it, it complained that there were missing (post compilation). Now I can't build it anymore.

Reply:
Does this only occur for aarch64-linux?

Reply:
On `aarch64-linux`, it doesn't even reach this stage:
```
-- Performing Test SUPPORTS_GC_SECTIONS - Success
-- Found Python3: /nix/store/1dv5m71dgl6n8dxd7riqjz094vqmyws8-python3-3.11.7/bin/python3.11 (found version "3.11.7") found components: Interpreter Development Development.Module Development.Embed 
MuJoCo is at /nix/store/m52vhg3hqsb6nay13ll1whvaf6bv11fm-mujoco-3.1.1/lib/libmujoco.so
MuJoCo headers are at /nix/store/m52vhg3hqsb6nay13ll1whvaf6bv11fm-mujoco-3.1.1/include
-- mujoco::FindOrFetch: checking for targets in package `absl`
-- mujoco::FindOrFetch: checking for targets in package `absl` - target `absl::core_headers` not defined.
-- mujoco::FindOrFetch: Using FetchContent to retrieve `abseil-cpp`
CMake Error at /nix/store/7jgkzsn972y4lrblv7f5rjpm46md0v64-cmake-3.27.8/share/cmake-3.27/Modules/ExternalProject.cmake:3176 (message):
  No download info given for 'abseil-cpp-populate' and its source directory:

   /build/mujoco-3.1.1/build/temp.linux-aarch64-cpython-311/_deps/abseil-cpp-src

  is not an existing non-empty directory.  Please specify one of:

   * SOURCE_DIR with an existing non-empty directory
   * DOWNLOAD_COMMAND
   * URL
   * GIT_REPOSITORY
   * SVN_REPOSITORY
   * HG_REPOSITORY
   * CVS_REPOSITORY and CVS_MODULE
Call Stack (most recent call first):
  /nix/store/7jgkzsn972y4lrblv7f5rjpm46md0v64-cmake-3.27.8/share/cmake-3.27/Modules/ExternalProject.cmake:4345 (_ep_add_download_command)
  CMakeLists.txt:29 (ExternalProject_Add)
```

Reply:
Cannot repro on `x86_64-linux` with https://github.com/tmplt/nixpkgs/commit/c480fd6bca1de0ed0ce3a7b089ff891c2f28ee78; but the python dep additions are ok: I get the same build error without them.

Are you getting this on a native `aarch64-linux` system or are you cross compiling?

Reply:
Can't repro on `aarch64-linux` either with
```
$ nix build github:tmplt/nixpkgs#legacyPackages.x86_64-linux.pkgsCross.aarch64-multiplatform.mujoco
```
Don't have the resources to try building the Python library; `pkgsCross` isn't in the binary cache.

Reply:
Removing them indeed breaks it:
```
Executing pythonRuntimeDepsCheck
Checking runtime dependencies for mujoco-3.1.1-cp311-cp311-linux_x86_64.whl
  - absl-py not installed
  - etils not installed
  - pyopengl not installed
```

And I still can't build at all on `aarch64-linux` I am using the nix-community builder and get [this message](https://github.com/NixOS/nixpkgs/pull/275195/files#r1451628196)

Reply:
How do you invoke the nix-community builder? I haven't heard about it.